client/src/components/Jobs/Jobs.jsx

```diff
@@ -137,27 +178,42 @@ function JobsView({ showDetails, showFancyStatus, rowProps }) {
   const filterColumns = columns.filter((f) => f.show !== false);
 
   return (
-    <InfiniteScrollTable
-      tableColumns={filterColumns}
-      tableData={jobs}
-      onInfiniteScroll={infiniteScrollCallback}
-      isLoading={isLoading}
-      className={showDetails ? 'jobs-detailed-view' : 'jobs-view'}
-      noDataText={noDataText}
-      getRowProps={rowProps}
-    />
+    <>
+      {includeSearchbar && (
+        <Searchbar
+          api="tapis"
+          resultCount={jobs.length}
+          dataType="Jobs"
+          infiniteScroll
+          disabled={isJobLoading || isNotificationLoading}
+        />
+      )}
+      <div className={includeSearchbar ? 'o-flex-item-table-wrap' : ''}>
+        <InfiniteScrollTable
```

Comment:
This works as is. That said, since we have access to the `jobs` data object, we could instead render `noDataText` directly when `!jobs.length`, which may allow us to make use of the flex container and center the message both horizontally and vertically. What do you think?

Reply:
I did have it structured this way in the first iteration which worked. Were you thinking of just moving just [this](https://github.com/TACC/Core-Portal/blob/2fdbccb387cc8933d1ac414d8300a3c2016665fd/client/src/components/Jobs/Jobs.jsx#L45-L47) additional piece I added or the entire noDataText component?

Reply:
The use of `<Section>` and `<SectionMessage>` are great additions for the `noDataText` component. Ideally, I think both strings (keyed off `query_string`) would render the same, just with different text.

The change I propose here might then be to conditionally render `<noDataText>` instead of `<InfiniteScrollTable>` when `!jobs.length`, while also incorporating the `<Section>` and `<SectionMessage>` change for both types of `noDataText`

Reply:
Yeah that makes sense to me. So we will essentially be sending nothing as part of the `InfiniteScrollTable` `noDataText` prop? Should I add that as part of this PR or create a separate task for it?

Reply:
Yeah essentially. That's a good point. It might be best for this to be done as a new task, and build in what I'm talking about here with the use of `Section` and `SectionMessage` into the `noDataText` prop such that it would be properly centered

Reply:
@rstijerina created task [TV3-135](https://jira.tacc.utexas.edu/browse/TV3-135)